Dockerfile

```diff
@@ -20,7 +20,7 @@
 # be world-readable!
 #
 #
-FROM mbrekkevold/navbase-debian:stretch
+FROM mbrekkevold/navbase-debian:buster
```

Comment:
IIRC, buster has python 3.7. Any reason to just move to 3.6?

Reply:
You raise a valid point :-)

Reply:
Changed Python 3.6 to Python 3.7.